app/index.js

```diff
@@ -251,30 +261,58 @@ smacssGenerator.prototype.copyCSSFiles = function copyCSSFiles() {
 
   // SMACSS - SCSS Structure
   // TODO: Update structure based on ticket #7
-  this.copy("scss/_master.scss", this.appName + "/app/scss/master.scss");
-  this.copy("scss/_base.scss", this.appName + "/app/scss/base.scss");
-  this.copy("scss/_layout.scss", this.appName + "/app/scss/layout.scss");
-  this.copy("scss/_reset.scss", this.appName + "/app/scss/reset.scss");
-  this.copy("scss/_variables.scss", this.appName + "/app/scss/variables.scss");
-  this.copy("scss/_mixins.scss", this.appName + "/app/scss/mixins.scss");
-  this.copy("scss/_module.scss", this.appName + "/app/scss/modules/module.scss");
-  this.copy("scss/_page_landing.scss", this.appName + "/app/scss/pages/page-landing.scss");
+  if(this.appType != 'typeAdminWebApp') {
+    this.copy("scss/_master.scss", this.appName + "/app/scss/master.scss");
+    this.copy("scss/_base.scss", this.appName + "/app/scss/base.scss");
+    this.copy("scss/_layout.scss", this.appName + "/app/scss/layout.scss");
+    this.copy("scss/_reset.scss", this.appName + "/app/scss/reset.scss");
+    this.copy("scss/_variables.scss", this.appName + "/app/scss/variables.scss");
+    this.copy("scss/_mixins.scss", this.appName + "/app/scss/mixins.scss");
+    this.copy("scss/_module.scss", this.appName + "/app/scss/modules/module.scss");
```

Comment:
Line is too long.<br>Mixed double and single quotes.<br>Too many errors. (59% scanned).